cluster/addons/kube-network-policies/OWNERS

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - bowei
+  - MrHohn
+emeritus_approvers:
+  - dnardo
+reviewers:
+  - bowei
+  - MrHohn
```

Comment:
(This file was obviously copied from somewhere else and is totally wrong)

Reply:
updated added you and me